color/src/color.rs

```diff
@@ -22,6 +22,7 @@ use crate::floatfuncs::FloatFuncs;
 /// for spline interpolation. For cylindrical color spaces, hue fixup should
 /// be applied before interpolation.
 #[derive(Clone, Copy, Debug)]
+#[repr(C)]
```

Comment:
I think `repr(transparent)` would be more accurate here; it would ensure that a new field being added requires more careful thought.

Reply:
Yes, I think `repr(transparent)` is right here. I wasn't sure `[f32; 4]` would be valid to cross the C ABI without the explicit `repr(C)`, but I think it is.